meson.build

```diff
@@ -0,0 +1,265 @@
+project('harfbuzz', 'c', 'cpp',
+  meson_version: '>= 0.47.0',
+  version: '2.2.0')
+
+pkgmod = import('pkgconfig')
```

Comment:
Please provide cmake module installation also, added on #822 to autotools, if possible

Reply:
Added something along those lines now. Produces same(ish) output as autotools, but it's all completely cargo culted.

Reply:
Thank you. It is not that vital obviously but we don't know about its current users number so was better to lose that capability :)